ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly.yaml

```diff
@@ -163,10 +163,22 @@ tests:
       ARCHITECTURE: arm64
       NVME_DEVICE: /dev/nvme1n1
       PACKET_PLAN: c3.large.arm64
-      TEST_ADDITIONAL: Network_Observability
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn
+- as: aws-ipi-f7-netobserv
```

Comment:
We have netobserv tests for amd64, and now we want to add tests for arm64, which means it supports both amd64 and arm64. Per the agreement in the team, if a test supports both amd64 and arm64, we move the test from amd64/arm64 to multi.  CC @jianlinliu

Reply:
Thanks for the awareness. Liang's statement is right, here is the jira epic [issue](https://issues.redhat.com/browse/OCPQE-17309) for the effort.

Reply:
Moved tests to multi